gen/Manifest.toml

```diff
@@ -16,21 +16,17 @@ version = "0.4.1"
 
 [[Clang]]
 deps = ["CEnum", "Clang_jll", "Downloads", "Pkg", "TOML"]
-git-tree-sha1 = "93b424759f11b7dbd300eb424fa1bfe590e15cf6"
-repo-rev = "master"
-repo-url = "https://github.com/JuliaInterop/Clang.jl.git"
+git-tree-sha1 = "59687b776ff47eebbab8422186b7eb670caf447f"
+repo-rev = "julia-1.7"
+repo-url = "https://github.com/JuliaInterop/Clang.jl"
 uuid = "40e3b903-d033-50b4-a0cc-940c62c95e31"
 version = "0.14.0"
 
 [[Clang_jll]]
 deps = ["Artifacts", "JLLWrappers", "Libdl", "Pkg", "libLLVM_jll"]
-git-tree-sha1 = "a5923c06de3178dd755f4b9411ea8922a7ae6fb8"
+git-tree-sha1 = "39e54fee89caf38de9196d8919475aa2ff65309d"
 uuid = "0ee61d77-7f21-5576-8119-9fcc46b10100"
-version = "11.0.1+3"
-
-[[CompilerSupportLibraries_jll]]
-deps = ["Artifacts", "Libdl"]
-uuid = "e66e0078-7015-5450-92f7-15fbd957f2ae"
+version = "12.0.0+1"
```

Comment:
Looks like Julia-1.7-beta4 will ship LLVM-12.0.1, so we may need to update this.

Reply:
This is just for generators right? Do those have to match any particular version?

Reply:
Actually it got removed by something? Not sure what happened